maintainers/maintainer-list.nix

```diff
@@ -14912,6 +14912,12 @@
     githubId = 496447;
     name = "Robert Hensing";
   };
+  robert-manchester = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.

Reply:
that is the order returned by sort, somewhat surprisingly, using various options (-d, -g, -h, no option).